src/Raven.Server/Documents/Handlers/ReplicationHandler.cs

```diff
@@ -236,10 +235,64 @@ public Task Performance()
             return true;
         }
 
+        [RavenAction("/databases/*/replication/pulses/live", "GET", AuthorizationStatus.ValidUser, SkipUsagesCount = true)]
+        public async Task PulsesLive()
```

Comment:
I won't call that endpoint replication pulses, as we want to have single web socket for pulses for:
replication (internal & external), sql etl, raven db etl